resources/iTerm2/readme.md

```diff
@@ -0,0 +1,29 @@
+# Remedy - Terminal
+
+> An iTerm 2 profile matching the colours used in VS Code's Remedy theme. 🎨
+
+## Installation
+
+- Download the _Remedy.itermcolors_ file from the [latest release assets][github-releases] on Github or from this directory
+- Launch iTerm 2
+- Click on **iTerm2** menu title
+- Select **Preferences...** option
+- Select **Profiles**
+- Navigate to **Colors** tab
+- Click on **Color Presets**
+- Click on **Import**
+- Select the _Remedy.itermcolors_ file
+- Click on **Color Presets** and choose the Remedy color scheme
+
+After the installation you will likely want to fix/change the following settings:
```

Comment:
I am not sure this section applies to iTerm. With a Terminal profile you have to export your complete Terminal configuration, including your chosen font, cursor appearance and other non-colour-related options but with iTerm I see only colours defined. Perhaps we can drop this?

Reply:
That's right, iTerm only exports colors. I left that section so that folks know what to change to make it look like the screenshot, and also to be consistent with the Terminal readme. So I would reword this into:

> After the installation you might want to customize the following settings:
> - Font (I use _Operator Mono_ with ligatures hacked into it 🤷‍)
> - Cursor (I' using 'Box')
> - Any other, non-colour-related settings

If you think it doesn't add much/doesn't apply, I can remove it!

Reply:
Sounds good! I would start the section with the following line, to emphasise that the next steps are only relevant if you want to make it look like in the screenshot.

> If you would like to make iTerm look like in the screenshot, you might want to customise the following settings:

(the rest as you wrote in the comment)

Reply:
Perfect! Fixed in e07d66a